cstore_fdw.c

```diff
@@ -300,6 +301,9 @@ CStoreProcessUtility(Node * parseTree, const char *queryString,
 		List *allTablesList = truncateStatement->relations;
 		List *cstoreTablesList = FindCStoreTables(allTablesList);
 		List *otherTablesList = list_difference(allTablesList, cstoreTablesList);
+
+		CheckTruncatePermissions(cstoreTablesList);
```

Comment:
In TruncateCStoreTable() in cstore_fdw.c, doesn't heap_open() acquire a lock and heap_close() release that lock?